app/controllers/wfc_controller.rb

```diff
@@ -8,17 +8,17 @@ def panel
   end
 
   def competition_export
-    select_attributes = [
-      :id, :name, :start_date, :end_date,
-      :country_id, :announced_at, :results_posted_at,
-      :currency_code, :base_entry_fee_lowest_denomination,
-      "count(distinct persons.id) as num_competitors"
+    select_attributes = %i[
+      id name start_date end_date
+      country_id announced_at results_posted_at
+      currency_code base_entry_fee_lowest_denomination
     ]
     from = params.require(:from_date)
     to = params.require(:to_date)
     response.headers["Content-Disposition"] = "attachment; filename=\"wfc-competitions-export-#{from}-#{to}.tsv\""
     @competitions = Competition
                     .select(select_attributes)
+                    .select('COUNT(DISTINCT persons.id) AS num_competitors')
```

Comment:
This new solution seems to me like it's exactly the same as the old solution. (Except that you wrote `COUNT` and `DISTINCT` in capital letters now). Is this change to this file strictly necessary?